tools/static-assets/skel-full/imports/api/links/methods.js

```diff
@@ -1,19 +1,24 @@
 // Methods related to links
 
 import { Meteor } from 'meteor/meteor';
-import { Links } from './links.js';
 import { check } from 'meteor/check';
+import { Links } from './links.js';
 
 Meteor.methods({
   'links.insert'(title, url) {
     check(url, String);
     check(title, String);
 
     // Check if this is a valid url
-    if (!url.match(/((http|https)\:\/\/)+[a-zA-Z0-9\.\/\?\:@\-_=#]+\.([a-zA-Z0-9\&\.\/\?\:@\-_=#])*/g)){
-      throw new Meteor.Error('Bad url. I.e https://www.meteor.com');
+    const re = /((http|https)\:\/\/)+[a-zA-Z0-9\.\/\?\:@\-_=#]+\.([a-zA-Z0-9\&\.\/\?\:@\-_=#])*/g;
```

Comment:
Honestly validating URLs doesn't seem like a good idea in general. Perhaps you could just check if it starts with `http` or something.